UnityProject/Assets/Scripts/Player/MovementV2/MovementSynchronisation.cs

```diff
@@ -1054,11 +1054,12 @@ public bool TryMove(ref MoveData newMoveData, GameObject byClient, bool serverPr
 			bool bumpedSomething = false;
 			if (Cooldowns.TryStart(playerScript, this, NetworkSide.Server))
 			{
-				foreach (var bump in Bumps)
+				for (int i = Bumps.Count - 1; i >= 0; i--)
```

Comment:
might be able to cause infinite loop idk

Reply:
yeah, changed it a bit just in case